core/src/chiplets/bitwise.rs

```diff
@@ -15,21 +15,15 @@ pub const OP_CYCLE_LEN: usize = 8;
 // --- OPERATION SELECTORS ------------------------------------------------------------------------
 
 /// Specifies a bitwise AND operation.
-pub const BITWISE_AND: Selectors = [Felt::ZERO, Felt::ZERO];
+pub const BITWISE_AND: Selectors = [Felt::ZERO];
```

Comment:
Does this need to be an array? Or can we get away with making it a simple `Felt`?

Reply:
Changed it to felt and did the corresponding refactoring.